package.json

```diff
@@ -19,10 +19,10 @@
   "license": "MIT",
   "repository": "github:nesk/puphpeteer",
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=9.0.0"
```

Comment:
Why is this required?

Reply:
https://github.com/puppeteer/puppeteer/releases/tag/v3.0.0
![image](https://user-images.githubusercontent.com/34161928/97175932-48010080-17a5-11eb-9ae1-dc12d0fb2647.png)